MarvelApiClient/CharactersApiClient.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  CharactersApiClient.swift
+//  MarvelApiClient
+//
+//  Created by Pedro Vicente on 14/11/15.
+//  Copyright © 2015 GoKarumi S.L. All rights reserved.
+//
+
+import Foundation
+import BrightFutures
+
+public class CharactersApiClient : BaseApiClient {
```

Comment:
In this context this is not related to a Repository patterns. I prefer to change the method names before to use DataSource as suffix. Any suggestion for the method names?